patroni/README.md

```diff
@@ -0,0 +1,109 @@
+# Patroni
+Patroni is a template for you to create your own customized, high-availability solution using Python and - for maximum accessibility - a distributed configuration store like ZooKeeper, etcd, Consul or Kubernetes. Database engineers, DBAs, DevOps engineers, and SREs who are looking to quickly deploy HA PostgreSQL in the datacenter-or anywhere else-will hopefully find it useful.
+
+## Installation
+
+<b>Note:</b> Check your AWS VSL location and change your region based on your configuration if the velero backup fails.
```

Comment:
I would remove this. This is only relevant if you are using Velero with snapshots.